docs/articles/ephemeral-environments.md

```diff
@@ -1,120 +1,84 @@
+- By storing these resources in Git and using a GitOps tool like ArgoCD or Flux to deploy them into your namespace.
 
 The below diagram describes this approach at a high level:
 
 ![Environment with Reused Agents](images/reused-agents-environment.png)
 
 ### Triggering provisioned resources
 
-Once your Testkube resources have been created in your ephemeral cluster and the agent has connected to its 
-Testkube Environment, you can trigger/use them in any way supported by Testkube. In case of Workflows, you 
-might use a GitOps approach to deploy [Execution CRDs](/articles/test-executions) to trigger their execution, 
-trigger Test Executions [from CI/CD](/articles/cicd-overview), or simply use/automate the 
+Once your Testkube resources have been created in your ephemeral namespace and the agent has connected to its
+Testkube Environment, you can trigger/use them in any way supported by Testkube. In case of Workflows, you
+might use a GitOps approach to deploy [Execution CRDs](/articles/test-executions) to trigger their execution,
+trigger Test Executions [from CI/CD](/articles/cicd-overview), or simply use/automate the
 Testkube CLI [run command](/cli/testkube_run) in your CI/CD pipeline.
 
 If your Testkube resources contain [Kubernetes Event Triggers](/articles/test-triggers), these will be triggered accordingly.
 
 ### Aggregating results
 
-The Testkube Dashboard will aggregate and show test execution results from each ephemeral cluster as its Testkube Agent 
-connects and executes its Workflows. 
+The Testkube Dashboard will aggregate and show test execution results from each ephemeral namespace as its Testkube Agent
+connects and executes its Workflows.
 
 :::info
 Testkube identifies execution results by the name of their Workflow, so as long as the Workflows have the same name
-across all your environments, they will be aggregated and shown in the Dashboard under the corresponding Workflow.
+across all your namespaces, they will be aggregated and shown in the Dashboard under the corresponding Workflow.
 :::
 
-## 2. Ephemeral Namespaces
-
-Instead of provisioning ephemeral clusters, it is not uncommon to use ephemeral namespaces within an existing cluster
-for the same purpose. Combined with Testkubes `executionNamespace` functionality you can in this scenario avoid
-having to install the Testkube Agent in each ephemeral namespace;
-
-- Install the Testkube Agent in a dedicated namespace (`testkube` by default).
-- Deploy Testkube Resources (Workflows, etc.) to the same namespace, possibly dynamically as part of your CI/CD Pipelines
-- Use the `executionNamespace` property in your Workflows to configure in which namespace they should execute,  
-  either statically or dynamically as shown below.
-- Trigger the execution of these Workflows as you would normally from CI/CD, Kubernetes Events, etc. 
-
-![Ephemeral Namespaces](images/ephemeral-namespaces.png)
-
-### Runtime `executionNamespace` configuration
```

Comment:
should we not mention this somehow?

Reply:
how do you mean? this doc isn't about provisioning ephemeral envs per se - its trying to show how TK can be used in ephemeral environments provisioned elsewhere!?